enhancements/security/file-integrity-operator.md

```diff
@@ -103,10 +117,23 @@ The operator is intended to be the sole controller of its operand resources (con
 
 ## Implementation History
 
-* Initial POC at https://github.com/mrogers950/file-integrity-operator
+* Current implementation: https://github.com/openshift/file-integrity-operator
+
+## Roadmap
+
+* Deploy daemonSets and configurations based on labels, in order to target specific MachineSets. Customers may have UPI RHEL workers on which they manage AIDE out-of-band, and only wish to target the RHCOS masters. MachineSets may require differing AIDE configurations.
+* Further secure AIDE process containers with SELinux to only allow writing to the DB and log files.
+* Enable the use of object storage for exposing the AIDE logs instead of configMaps. See the Drawbacks section for more regarding AIDE log storage.
+* Add a "Provider" field to the API and enable other integrity checking mechanisms.
+  * Enable non-userspace and strongly attested integrity checking mechanism(s). Some suggested candidates include fs-verity and IMA. Note: Both IMA/fs-verity methods provide more integrity assurance over AIDE _only_ when backed by a TPM.
+    * fs-verity: According to https://www.kernel.org/doc/html/latest/filesystems/fsverity.html, fs-verity works only on read-only files. Requires ext4 formatting options and block size to be == PAGE_SIZE. Files need to be ioctl'ed as verity files and have signature verification performed in userspace. Unless fs-verity is backed by attested dm-verity volumes (not feasible for nodes) or tied into IMA somehow, this provider seems insufficient for our use.
```

Comment:
Well, we'll have to have a debate about that :smile:
But it doesn't have to happen now.

Reply:
No problem :) 
This was just my general assessment after reading the documentation. If there's ultimately a way we can leverage fs-verity (perhaps only together with IMA ?) that is all good with me.